shapes/Circle.cpp

```diff
@@ -6,6 +6,11 @@ Circle::Circle(double r)
     : r_(r)
```

Comment:
no constructor delegation. It takes place when one constructor calls another of the same class on initializer list.